### ISSUE TITLE
Provide more info?

### ISSUE THREAD

**Issue by jaandrle** [jaandrle/randomMDN#4]
Investigate JSON format https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand/index.json

Ideas:
- [ ] browserCompat[^bc_repo][^bc_json]
- [ ] examle(s) (screenshot)
- [ ] detect experimental (`/class="notecard experimental"/`)
- [ ] …

Investigation…
```bash
nodejsscript --print '(await fetch("https://developer.mozilla.org/en-US/docs/Web/API/Document/createCDATASection/index.json").then(res=> res.json())).doc.parents.map(o=> o.title)'

nodejsscript --print '(await fetch("https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics/fontBoundingBoxDescent/index.json").then(res=> res.json())).doc.sidebarHTML.matchAll(/<strong>(.*?)<\/strong>/g)' 'a=> Array.from(a).map(([ _, m ])=> m)'
```

[^bc_json]: https://unpkg.com/@mdn/browser-compat-data@5.2.62/data.json
[^bc_repo]: https://github.com/mdn/browser-compat-data

**Comment by jaandrle** [comment]
Without much change, it could be added [Baseline (compatibility)](https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility), it is just `baseline` key in json:


1. example without provided baseline: https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility/index.json (glossary-definition), https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onUpdateAvailable/index.json (webextension-api-event)
1. example of limited availability: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture/index.json
1. example of newly available: https://developer.mozilla.org/en-US/docs/Web/CSS/cos/index.json
1. example of widely available: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError/errors/index.json

…emojis
- 🦖 no baseline: ![Screenshot 2024-12-18 at 11-33-38 Home - Front-End Social](https://github.com/user-attachments/assets/682f8cbc-01b9-46a3-a053-393ced4b5a64)
- 🟧 limited availability: ![Screenshot 2024-12-18 at 11-23-55 randomMDN (@randomMDN@front-end social) - Front-End Social](https://github.com/user-attachments/assets/9499a8e9-aaf6-4488-b3d7-23a64450f630)
- ☑️ newly: ![Screenshot 2024-12-18 at 11-28-36 Home - Front-End Social](https://github.com/user-attachments/assets/d332f312-ff7c-45ba-acbd-c34c202e5560)
- ✅ widely: ![Screenshot 2024-12-18 at 11-18-38 randomMDN (@randomMDN@front-end social) - Front-End Social](https://github.com/user-attachments/assets/100a3000-1dc0-4c1b-9fe7-ec2ef5b1f702)

**Known issue**:
1. The information is static based on tooted date (there can be weird situation when the next day the status differ)

**Comment by rauschma** [comment]
I’m wondering how many people know the symbols. Maybe you can add text? Then I’d put the information in a separate line:

✅ Widely available
☑️ Newly available 2022
🔶 Limited availability
❌ No baseline

* Only “newly available” seems to have a year. Not sure why.
* Unclear: Should the year be put at the end or after the emoji (before the text)
* My source: https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility

<blockquote><img src="https://developer.mozilla.org/mdn-social-share.d893525a4fb5fb1f67a2.png" width="48" align="right"><div><img src="https://developer.mozilla.org/favicon-48x48.bc390275e955dacb2e65.png" height="14"> MDN Web Docs</div><div><strong><a href="https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility">Baseline (compatibility) - MDN Web Docs Glossary: Definitions of Web-related terms | MDN</a></strong></div><div>Baseline identifies the availability of web platform features across popular browsers, including APIs, CSS properties, and JavaScript syntax. Baseline describes web features as being either widely available or newly available. Features that do not meet the Baseline criteria are said to have limited availability.</div></blockquote>

**Comment by jaandrle** [comment]
Hmm, I was a bit worried about the character count, but looking at recent posts, there tends to be a buffer of ~100 characters.

TLDR, I guess I wouldn't mind putting it on the next line with a description. On the other hand, it could be “unnecessarily” repetitive and the description could be in the bot profile. What do you think?

Regarding year. The baseline (compatibility) is what determines the deployment of these things in production. So, for new stuff, you can get an indication of the year when it will be Widely available.

Articles without a baseline probably can't have ❌. From what I've checked, most likely only the various overviews, help and guide pages don't have it filled in. Just like the one about Baseline compatibility. So in that case I would either not list it at all or use something neutral.
